### PR TITLE
Allow specifying the min and max version number in qrCreate

### PR DESCRIPTION
**Pull request by livecodepanos**
Submitting this PR on behalf of Matthias Rebbe

### CODE REVIEW

File: extensions/script-libraries/qr/qr.livecodescript [hunk 1]
```diff
@@ -1522,7 +1559,13 @@ command qrCreate pImg, pData, pECC, pSize, pMask
       # calculate minimum version required
       local tMinVersion
       put 0 into tMinVersion
-      repeat with tVersion = 1 to 40
+      
+      if pMinVersion is empty then put 1 into pMinVersion
```
Comment (montegoulding) [inline extensions/script-libraries/qr/qr.livecodescript]: @livecodepanos I realise it's not your code but is there any chance you could change all these to multi-line `if`? The single line form rubs me up the wrong way.

**Comment by livecodepanos** [comment]
@montegoulding Sure, done :)

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 46a6462

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 46a64620798332f3c0018c2f686cefda1b71aa92

**Comment by livecode-vulcan** [comment]
😎 test success 46a64620798332f3c0018c2f686cefda1b71aa92

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/170)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/162)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/162)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/162)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/162)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1684)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/66)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/66)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1714)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1707)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1704)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1740)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1681)

**Pull request merged**